### PR TITLE
Rename ProsessJarFiles project to DownloadDependencies

### PR DESCRIPTION
**Pull request by ugras-ergun-sonarsource**
Part of #5262

### CODE REVIEW

File: build/DownloadDependencies/DownloadDependencies.proj [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+       So, to preserve the formatting we'll put most of the logic in files that we import. -->	
+  <PropertyGroup>	
+    <ProjectGuid>{3ECCAF9D-3B23-4980-83E2-8ACEE0FC6BEB}</ProjectGuid>	
+  </PropertyGroup>	
+  <Import Project="$(MSBuildThisFileDirectory)CommonMSBuild.targets" />	
+  <Import Project="$(MSBuildThisFileDirectory)CommonProperties.props" />	
+  <Import Project="$(MSBuildThisFileDirectory)JarProcessing.targets" />	
+  <ItemGroup>	
+    <None Include="CommonProperties.props" />	
+    <None Include="JarProcessing.targets" />
```
Comment (georgii-borovinskikh-sonarsource) [inline build/DownloadDependencies/DownloadDependencies.proj:12]: duplication?

File: build/DownloadDependencies/DownloadDependencies.proj [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  </PropertyGroup>	
+  <Import Project="$(MSBuildThisFileDirectory)CommonMSBuild.targets" />	
+  <Import Project="$(MSBuildThisFileDirectory)CommonProperties.props" />	
+  <Import Project="$(MSBuildThisFileDirectory)JarProcessing.targets" />	
+  <ItemGroup>	
+    <None Include="CommonProperties.props" />	
+    <None Include="JarProcessing.targets" />	
+  </ItemGroup>	
+  <ItemGroup>	
+    <ProjectReference Include="..\tools\ExtractRuleDescFromJson\ExtractRuleDescFromJson.csproj">
```
Comment (georgii-borovinskikh-sonarsource) [inline build/DownloadDependencies/DownloadDependencies.proj:15]: This needs to be deleted

**Review by georgii-borovinskikh-sonarsource — APPROVED** [review]

File: build/DownloadDependencies/DownloadDependencies.proj (hunk 1 above)
Comment (ugras-ergun-sonarsource) [reply build/DownloadDependencies/DownloadDependencies.proj:12]: no it's to see on visual studio.

File: build/DownloadDependencies/DownloadDependencies.proj (hunk 2 above)
Comment (ugras-ergun-sonarsource) [reply build/DownloadDependencies/DownloadDependencies.proj:15]: This is just rename of project, cleanup will be part of another PR

**Pull request merged**